src/components/analyses/landing/AnalysisSubmissionLanding.js

```diff
@@ -0,0 +1,620 @@
+        mutate: analysisCommentMutation,
+        isLoading: commentLoading,
+        error: commentError,
+    } = useMutation(updateAnalysisComment, {
+        onSuccess: (analysis) => {
+            refreshAnalysis();
+            setCommentDialogOpen(false);
+        },
+    });
+
+    const onShareWithSupport = (analysis, comment) => {
+        shareAnalysesMutation({
+            ...getAnalysisShareWithSupportRequest(
+                config?.analysis?.supportUser,
+                analysis.id
+            ),
+            supportRequest: analysisSupportRequest(
+                userProfile?.id,
+                userProfile?.attributes.email,
+                t("statusHelpRequestSubject", {
+                    name: userProfile?.attributes.name,
+                }),
+                comment,
+                analysis
+            ),
+        });
+    };
+
+    const { mutate: doTimeLimitExtension, isLoading: extensionLoading } =
+        useMutation(extendVICEAnalysisTimeLimit, {
+            onSuccess: (resp) => {
+                setConfirmExtendTimeLimitDlgOpen(false);
+                setTimeLimit(null);
+                //convert the response from seconds to milliseconds
+                announce({
+                    text: t("timeLimitExtended", {
+                        newTimeLimit: formatDate(resp?.time_limit * 1000),
+                        analysisName: analysis?.name,
+                    }),
+                    variant: SUCCESS,
+                });
+            },
+            onError: (error) => {
+                showErrorAnnouncer(t("analysesRelaunchError"), error);
+            },
+        });
+
+    const handleCancel = () => {
+        analysesCancelMutation({ id: analysis?.id });
+    };
+
+    const handleSaveAndComplete = () => {
+        const id = analysis?.id;
+        analysesCancelMutation({
+            id,
+            job_status: analysisStatus.COMPLETED,
+        });
+    };
+
+    const busy =
+        isFetching ||
+        isInfoFetching ||
+        isParamsFetching ||
+        analysisLoading ||
+        isFetchingTimeLimit ||
+        extensionLoading;
+
+    if (busy) {
+        return <GridLoading rows={25} baseId={baseId} />;
+    }
+
+    if (analysisFetchError) {
+        return (
+            <WrappedErrorHandler
+                errorObject={analysisFetchError}
+                baseId={baseId}
+            />
+        );
+    }
+
+    //if the analysis not found the a give id, analysis will be null/undefined.
+    if ((analysis === null || analysis === undefined) && !busy) {
+        return <ErrorTypography errorMessage={t("analysisNotFound")} />;
+    }
+
+    return (
+        <PageWrapper appBarHeight={75}>
+            <div
+                style={{ margin: theme.spacing(1), padding: theme.spacing(1) }}
+            >
+                {view === BATCH_DRILL_DOWN && <BackButton />}
+                <Grid container spacing={1}>
+                    <Grid item xs={isMobile ? 0 : 6}>
```

Comment:
i think it can be some thing  like  `<Grid` item xs={12} sm={6}>`. I got this from - https://v4.mui.com/components/grid/#grid-with-breakpoints

Reply:
But the logic here looks like `<Grid item xs={0} sm={6}>` to me, and according to https://v4.mui.com/api/grid/, the `xs` prop defaults to `false`, which I'm guessing is treated the same as `xs={0}`?

So I think `<Grid item sm={6}>` is the same as `<Grid item xs={0} sm={6}>`, which is the same as `<Grid item xs={isMobile ? 0 : 6}>` (I think, since `isMobile = useMediaQuery(theme.breakpoints.down("xs"))`).